test/clj/rems/circle_ci.clj

```diff
@@ -0,0 +1,49 @@
+(ns rems.circle-ci
+  "Kaocha plugin that allows filtering created test plan before execution with testable ids.
+   Allows using CircleCI test splitting for parallel tests, e.g. slow browser tests.
+
+   Heavily inspired by https://andreacrotti.github.io/2020-07-28-parallel-ci-kaocha/"
+  (:require [clojure.java.shell :as sh]
+            [clojure.string :as str]
+            [kaocha.plugin :as p]
+            [kaocha.testable :refer [test-seq]]
+            [medley.core :refer [update-existing]]))
+
+(defn get-test-ids [test-plan]
+  (->> (test-seq test-plan)
+       (sequence (comp (mapcat :kaocha.test-plan/tests)
+                       (remove :kaocha.testable/skip)
+                       (filter (comp #{:kaocha.type/var} :kaocha.testable/type))
+                       (map :kaocha.testable/id)))))
+
+(defn split-test-ids [test-ids]
+  (->> (sh/sh "circleci" "tests" "split" "--split-by=timings"
+              :in (str/join "\n" test-ids))
+       :out
+       (str/split-lines)
+       (into #{} (map #(keyword (subs % 1)))))) ; str/join turns :a/b into ":a/b", but keyword prepends ":", so remove first
```

Comment:
I don't understand this explanation here.

Reply:
this has been changed now, but original intent was to explain that the CLI command returns list of keywords as strings like `":rems.test-browser.test-new-application\n:rems.test-browser.test-handling"`, which needs some preprocessing:
```clojure
(keyword ":rems.test-browser.test-handling") ;=> ::rems.test-browser.test-handling
(keyword "rems.test-browser.test-handling") ;=> :rems.test-browser.test-handling
```